### PR TITLE
add ability to filter by app

### PR DESCRIPTION
**Pull request by jharewinton**
ended up being painfully simple to do, after going on several rubbish tangents

### CODE REVIEW

File: project/index.html [hunk 1]
```diff
@@ -8,6 +8,11 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.min.css">
     <link rel="stylesheet" href="/public/styles.css" type="text/css">
 
+    <script>
+      var appParam = window.location.search.split("=")[1];
```
Comment (gtrufitt) [inline project/index.html]: Maybe use the [urlSearchParams spec](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams): 

```
var queryParams = new URLSearchParams(window.location.search);
var appParam = queryParams.get('app');
```

File: project/index.html [hunk 2]
```diff
@@ -8,6 +8,11 @@
...
+      console.log("Application filter = " + appParam);
```
Comment (gtrufitt) [inline project/index.html]: Can probably remove this `log`?

**Review by gtrufitt — APPROVED** [review]
👍

File: project/index.html (hunk 1 above)
Comment (jharewinton) [reply project/index.html]: I wanted to, but a large chunk of editorial machines don't run recent enough browsers http://caniuse.com/#search=URLSearchParams

File: project/index.html (hunk 2 above)
Comment (jharewinton) [reply project/index.html]: 👍

**Pull request merged**